backend/migrations/versions/442dd8f223a9__alembic_sync.py

```diff
@@ -1,34 +1,36 @@
 """_describe_changes_shortly
 
-Revision ID: c72d8c5272a8
+Revision ID: 442dd8f223a9
 Revises: f6cd4ba7dd8d
-Create Date: 2024-02-08 12:35:45.016427
+Create Date: 2024-02-08 13:19:22.898697
 
 """
 from alembic import op
 import sqlalchemy as sa
 from sqlalchemy.dialects import postgresql
 
 # revision identifiers, used by Alembic.
-revision = 'c72d8c5272a8'
+revision = '442dd8f223a9'
 down_revision = 'f6cd4ba7dd8d'
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.drop_table('item_tags')
-    op.drop_table('tag')
-    op.drop_table('redshiftcluster_datasettable')
-    op.drop_table('redshiftcluster')
     op.drop_table('dataset_table_profiling_job')
+    op.drop_table('group_member')
+    op.drop_table('redshiftcluster')
     op.drop_table('dataset_quality_rule')
+    op.drop_table('tag')
+    op.drop_table('redshiftcluster_datasettable')
+    op.drop_table('item_tags')
     op.drop_table('redshiftcluster_dataset')
-    op.drop_table('group_member')
     op.alter_column('datapipelineenvironments', 'region',
```

Comment:
We should fix the original table, this field should not be updated to be nullable. Same for line 36 and 40